CoE_jump_start/config.yaml

```diff
@@ -1,10 +1,19 @@
 api: sambastudio
-llm:
-  temperature: 0.1
-  max_tokens_to_generate: 1024
-  sambaverse_model_name: "Meta-Llama-3-70B-Instruct"
-  samabaverse_select_expert: "Meta-Llama-3-70B-Instruct"
-  coe_routing: true
+
+llm: 
+    "temperature": 0.0
+    "do_sample": False
+    "max_tokens_to_generate": 1200
+    "sambaverse_model_name": "Meta/Meta-Llama-3-8B-Instruct"
+    "coe": True #set as true if using Sambastudio CoE endpoint
+    "select_expert": "Meta-Llama-3.1-70B-Instruct" #set if using FastCoE, Sambaverse or SambaStudio CoE llm expert
+    #fastApi CoE expert name -> "llama3-8b"
```

Comment:
this kit is not compatible with fastapi, i think we should remove this line

Reply:
Now removed thanks @snova-petrojm